person.rb

```diff
@@ -24,4 +25,9 @@ def of_age?
   def validate_name
     @name = @corrector.correct_name(@name)
   end
+
+  def add_rental(rental)
+    @rentals << rental
+    rental.person = self unless rental.person == self
+  end
```

Comment:
Please make `#add_rental` public as it would be accessed outside of the class. A private method can only be called within the same class.